PowerballApi/Controllers/PowerballController.cs

```diff
@@ -28,5 +31,28 @@ public IHttpActionResult PowerballResults()
 
 			return Ok(response);
 		}
+
+		[HttpGet]
+		[Route("drawings/{id}")]
+		public IHttpActionResult PowerballResult(string id)
+		{
+			var response = _repository.Get(id);
+
+			if (response == null)
+				return NotFound();
+
+			return Ok(response);
+		}
+		[HttpGet]
+		[Route("drawings")]
+		public IHttpActionResult PowerballResults(string after, string before)
```

Comment:
Given these parameters, there's now an inconsistency between these params and the params accepted by the repository method. I've created a task to clean this up later.